data/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Additional Funding from #startsmall"
+date: '2024-05-04'
+tags: ['OpenSats', 'funding', 'donations', 'startsmall', 'bitcoin', 'nostr']
+images: ['/static/images/blog/32-startsmall.jpg']
+draft: false
+summary: "OpenSats receives $21 million from #startsmall for bitcoin, nostr, and related FOSS development."
+---
+
+
+We are thrilled to announce that OpenSats has received an additional donation of
+\$21M from [#startsmall](https://startsmall.llc/), Jack Dorsey's philanthropic
+initiative. This donation will top up our funds and operations budget, with
+\$15,000,000 going to our [General Fund][gf], \$5,000,000 to [The Nostr
+Fund][nf], and \$1M to our [operations budget][ops].
+
+> Bitcoin changes absolutely everything. I don't think there is anything more
+> important in my lifetime to work on.
+>
+><cite>—[jack][JD]</cite>
+
+We agree, and are honored and humbled to receive such a generous donation. We
+will do our best to continually allocate the funds to high-impact projects and
+[freedom tech infrastructure](/blog/bitcoin-infrastructure-this-decade).  As of
+this writing, OpenSats has allocated close to [\$10,000,000 in grants][EOY2023],
+supporting over 100 grantees, many of whom for the [long term](/tags/lts).
+
+As a 501(c)(3) public charity, we rely on donations from the general public,
+including philanthropic organizations such as #startsmall and the [Reynolds
```

Comment:
```suggestion
including philanthropic organizations such as #startsmall and the [Reynolds
```
including philanthropic organizations such as [#startsmall](https://startsmall.llc/) and the [Reynolds

Reply:
Might want to link to #startsmall again?

Reply:
+1